src/Skoruba.IdentityServer4.Admin/Constants/AuthorizationConsts.cs

```diff
@@ -5,16 +5,17 @@ namespace Skoruba.IdentityServer4.Admin.Constants
     public class AuthorizationConsts
     {
         public const string AdministrationPolicy = "RequireAdministratorRole";
-        public const string AdministrationRole = "SkorubaIdentityAdminAdministrator";
-
-        public const string IdentityAdminCookieName = "IdentityServerAdmin";        
-        public const string IdentityAdminRedirectUri = "http://localhost:9000/signin-oidc";
-        public const string IdentityServerBaseUrl = "http://localhost:5000";
-        public const string IdentityAdminBaseUrl = "http://localhost:9000";
+        
+        // (cerg): moved to IAdminAppSettings
```

Comment:
I think, it will be better remove this commented lines.

Reply:
No problem, i will remove them